101-datastore-create-psql/azuredeploy.json

```diff
@@ -0,0 +1,104 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"workspaceName": {
+			"type": "string",
+			"metadata": {
+				"description": "Specifies the name of the Azure Machine Learning workspace which will hold this datastore target."
+			}
+		},
+		"datastoreName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of the datastore, case insensitive, can only contain alphanumeric characters and underscore"
+			}
+		},
+		"serverName": {
+			"type": "string",
+			"metadata": {
+				"description": "The PostgreSQL server name."
+			}
+		},
+		"databaseName": {
+			"type": "string",
+			"metadata": {
+				"description": "The database name."
+			}
+		},
+		"userId": {
+			"type": "string",
+			"metadata": {
+				"description": "The user ID."
+			}
+		},
+		"password": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The password."
+			}
+		},
+		"port": {
+			"type": "string",
+			"defaultValue": "5432",
+			"metadata": {
+				"description": "Optional : The port number. Defaults to 5432"
+			}
+		},
+		"endpoint": {
+			"type": "string",
+			"defaultValue": "postgres.database.azure.com",
```

Comment:
see comments in other PRs and make the same changes here

Reply:
looks like the changes have not been pushed here...

Reply:
Pushed, sorry I missed to push from local.